test/android/core/javatests/src/org/xwalk/core/xwview/test/ShouldOverrideUrlLoadingTest.java

```diff
@@ -469,7 +471,7 @@ public Boolean call() throws Exception {
             }
         });
 
-        assertEquals(1, mShouldOverrideUrlLoadingHelper.getCallCount());
+        assertEquals(shouldOverrideUrlLoadingCallCount, mShouldOverrideUrlLoadingHelper.getCallCount());
```

Comment:
Personally I prefer using a number here since it's easier to just look at a zero and say "hmm, so there shouldn't be any calls here". But I won't object if you prefer to keep it this way.